common/src/main/java/fr/rakambda/fallingtree/common/tree/TreeHandler.java

```diff
@@ -64,13 +64,35 @@ public BreakTreeResult breakTree(@NotNull ILevel level, @NotNull IPlayer player,
 		}
 		catch(TreeTooBigException e){
 			mod.notifyPlayer(player, mod.translate("chat.fallingtree.tree_too_big", mod.getConfiguration().getTrees().getMaxScanSize()));
-			throw new TreeBreakingException(e);
+			return BreakAbortionCause.TREE_TOO_BIG_SCAN;
 		}
 		catch(BreakTreeTooBigException e){
 			mod.notifyPlayer(player, mod.translate("chat.fallingtree.break_tree_too_big", mod.getConfiguration().getTrees().getMaxSize()));
-			throw new TreeBreakingException(e);
+			return BreakAbortionCause.TREE_TOO_BIG_BREAK;
 		}
 	}
+
+	@Deprecated
+	@NotNull
+	public BreakTreeResult breakTree(@NotNull ILevel level, @NotNull IPlayer player, @NotNull IBlockPos blockPos) throws TreeBreakingNotEnabledException, PlayerNotInRightState, ToolUseForcedException, TreeBreakingException, NoTreeFoundException, NotServerException{
```

Comment:
Probably kept it to do comparaisons? Can de deleted and rename the new `attemptTreeBreaking` to `breakTree`

Reply:
It was kept in order to preserve ABI. Deleted it since that doesn't seem to be an issue